File: LeetCode/Easy/1-two-sum/try-1.py
/*
 * Problem #1: Two Sum
 * Difficulty: Easy
 * Submission: Try 1
 * status: Accepted
 * Language: python
 * Date: 4/5/2026, 9:01:40 AM
 * Link: https://leetcode.com/problems/two-sum/
 */

class Solution(object):
    def twoSum(self, nums, target):
        d = {}  # value : index
        
        for i, num in enumerate(nums):
            complement = target - num
            
            if complement in d:
                return [i, d[complement]]
            
            d[num] = i
        

File: LeetCode/Easy/66-plus-one/try-1.java
/*
 * Problem #66: Plus One
 * Difficulty: Easy
 * Submission: Try 1
 * status: Accepted
 * Language: java
 * Date: 7/22/2026, 8:26:48 PM
 * Link: https://leetcode.com/problems/plus-one/
 */

class Solution {
    public int[] plusOne(int[] digits) {
        for (int i = digits.length - 1; i >= 0; i--) {
            if (digits[i] < 9) {
                digits[i]++;
                return digits;
            }
            digits[i] = 0;
        }

        int[] result = new int[digits.length + 1];
        result[0] = 1;
        return result;
    }
}
  
